async_simple/coro/test/LazyTest.cpp

```diff
@@ -448,7 +448,7 @@ TEST_F(LazyTest, testCollectAllBatched) {
         EXPECT_EQ(task_num, out.size());
         co_await CurrentExecutor();
         int sum = 0;
-        for (auto i = 0; i < out.size(); i++) {
+        for (size_t i = 0; i < out.size(); i++) {
```

Comment:
Is this necessary? Does GCC warn if we use `auto` here?

Reply:
`auto` is deduced to `int` here.

Reply:
Got it. We'd better use range-based for here. But we could do it in next patches.